keyboards/0xcb/1337/config.h

```diff
@@ -47,7 +47,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 #define RGB_DI_PIN D3
 #ifdef RGB_DI_PIN
-  #define RGBLED_NUM 5
+  #define RGBLED_NUM 4
```

Comment:
This shouldn't be changed here.  This should be changed in your keymaps config.h.

Reply:
But the pcb only has 4 rgb LEDs why is it keymap specific?

Reply:
I put 5 in there by mistake